features/api/turnkey_projects.feature

```diff
@@ -28,30 +28,34 @@ Feature:
       "category":"Nature et Environnement",
       "title":"Stop mégots !",
       "slug":"stop-megots",
-      "subtitle":"Campagnes de sensibilisation et de revalorisation des mégots jetés"
+      "subtitle":"Campagnes de sensibilisation et de revalorisation des mégots jetés",
+      "position": 1
```

Comment:
why do we need to have `position` field in the response, the ordered list (without position field) is not enough?

Reply:
Yes, there is no need of position. I removed it.